src/protocol/generate-python.py

```diff
@@ -76,11 +77,20 @@ def is_vector(payload):
 if not os.path.exists(output_path):
     os.makedirs(output_path)
 
+try:
```

Comment:
If I for the idea right, this is not working, the script is not able to change the permissions here. 
```
 patrick@dell  /mnt/hd/git/blue/ping-protocol-cpp/src/protocol   python-update  ./generate-python.py                                                   
 patrick@dell  /mnt/hd/git/blue/ping-protocol-cpp/src/protocol   python-update  ./generate-python.py
Traceback (most recent call last):
  File "./generate-python.py", line 93, in <module>
    f = open(os.path.join(output_path, "Ping1D.py"), "w")
PermissionError: [Errno 13] Permission denied: '/mnt/hd/git/blue/ping-protocol-cpp/src/protocol/python/Ping1D.py'

```